.env.sample

```diff
@@ -1 +1,18 @@
-GRAPH_URI=http://localhost:4000/graphiql
\ No newline at end of file
+GRAPH_URI=
+ONE_SIGNAL_APP_ID=
+ONE_SIGNAL_ACCOUNT_TAG_SALT=
+WALLET_CONNECT_PROJECT_ID=
+WALLET_CONNECT_METADATA_URL=
+MAINNET_RPC_URL=
+DEVNET_RPC_URL=
+SOLANA_PREVIEW=
```

Comment:
is this one still needed?

Reply:
this whole file?

Reply:
He's talking about the SOLANA_PREVIEW var